mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -37,6 +37,31 @@ import org.apache.spark.sql.types._
  */
 private[feature] trait RFormulaBase extends HasFeaturesCol with HasLabelCol {
 
+  /**
+   * Param for how to order labels of string column. The first label after ordering is assigned
```

Comment:
Please mentions of ```label/labels``` in all comments, since we handle all string columns regardless of feature columns or label column. The current description may confused users.